rust/sbp/src/messages/gnss.rs

```diff
@@ -259,14 +255,15 @@ impl crate::serialize::SbpSerialize for GPSTimeSec {
 
 /// Represents all the relevant information about the signal
 ///
-/// Signal identifier containing constellation, band, and satellite identifier
+/// Signal identifier containing constellation, band, and satellite
+/// identifier.
 ///
 #[cfg_attr(feature = "sbp_serde", derive(serde::Serialize))]
 #[derive(Debug, Clone)]
 #[allow(non_snake_case)]
 pub struct GnssSignal {
     /// Constellation-specific satellite identifier. This field for Glonass can
-    /// either be (100+FCN) where FCN is in [-7,+6] or  the Slot ID in [1,28]
+    /// either be (100+FCN) where FCN is in [-7,+6] or the Slot ID in [1,28].
```

Comment:
I'm going to shamelessly steal the suggestions from `cargo doc --open`

```
warning: unresolved link to `1,28`
   --> rust/sbp/src/messages/gnss.rs:266:72
    |
266 |     /// either be (100+FCN) where FCN is in [-7,+6] or the Slot ID in [1,28].
    |                                                                        ^^^^ no item named `1,28` in scope
    |
    = note: `#[warn(broken_intra_doc_links)]` on by default
    = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`
```

Reply:
These are https://swift-nav.atlassian.net/browse/INFRA-485

Reply:
Sorry I missed that.